torch/functional.py

```diff
@@ -347,6 +347,14 @@ def einsum(*args):
         run faster or consume less memory. Projects like opt_einsum (https://optimized-einsum.readthedocs.io/en/stable/)
         can optimize the formula for you.
 
+    .. note::
+
+        As of PyTorch 1.9 :func:`torch.einsum` also supports the sublist format (see examples below). In this format,
```

Comment:
The second and third sentences seem largely redundant.

What about:

"In this format, subscripts for each operand are specified by a "sublist" of integers in the range [0, 52). These sublists follow their operands, and an extra sublist can appear at the end of the input to specify the output's subscripts."

?